lib/internal/Magento/Framework/Mview/View/ChangelogInterface.php

```diff
@@ -11,6 +11,11 @@
  */
 interface ChangelogInterface
 {
+    const ATTRIBUTE_SCOPE_SUPPORT = 'attribute_scope';
+    const STORE_SCOPE_SUPPORT = 'store_scope';
+    const ATTRIBUTE_COLUMN = 'attribute_id';
+    const STORE_COLUMN = 'store_id';
```

Comment:
Better to name this more agnostic like "scope_id"
Different attributes can have different scope (global, store, website level)